tests/math/test_is_independent.py

```diff
@@ -252,13 +255,8 @@ def test_get_random_args(self, args, num, bounds):
 
     dev = qml.device("default.qubit", wires=1)
 
-    @qml.qnode(dev, interface=interface)
-    def const_circuit(x, y):
-        qml.RX(0.1, wires=0)
-        return qml.expval(qml.PauliZ(0))
-
     constant_functions = [
-        const_circuit,
+        qml.QNode(const_circuit, dev, interface=interface),
         lambda x: np.arange(20).reshape((2, 5, 2)),
         lambda x: (np.ones(3), -0.1),
```

Comment:
funny that pylint doesn't complain about the unused `x`'s here

Reply:
Good point :sweat_smile: